tests/foreman/rhai/test_rhai.py

```diff
@@ -130,6 +131,9 @@ def test_unregister_system_from_rhai(self):
                 session.nav.click(
                     locators['insights.unregister_button']
                 )
+                self.browser.refresh()
+                time.sleep(30)
```

Comment:
@elyezer I understand that point and tried using all the options, before picking up time.sleep: 
1. self.browser.refresh() only
2. self.browser.refresh and wait_for_ajax()
3.self.browser.refresh() and time.sleep(30)
The first two did not work and thus decided to go with time.sleep()
Here is the output[1] with option 2 where the test fails.
[1] http://pastebin.test.redhat.com/329618